.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,298 @@
+eclipse.preferences.version=1

```

Comment:
I have remember out how to delete them from git without deleting them locally. As I have said, it has been some years ...